backend/src/services/group/group.service.ts

```diff
@@ -40,6 +41,12 @@ class Group {
     groupsRequestDto: GroupsCreateRequestDto,
   ): Promise<GroupsResponseDto> {
     const { name, permissionIds, userIds } = groupsRequestDto;
+    const groupByName = await this.#groupsRepository.getByName(name);
+
+    if (groupByName) {
+      throw new InvalidGroupError();
+    }
```

Comment:
```suggestion
    if (groupByName) {
      throw new GroupsError();
    }
```
errors should be per service